workflows/ZWEWRF0n.xml

```diff
@@ -433,5 +433,37 @@ exit $?</inlineTemplate>
             <!-- zosmfOutput -->
         </template>
     </step>
-    <!-- TODO: Optionally start the STC -->
+    <step name="run_stc" optional="false">
+        <title>Start the Zowe instance</title>
+        <description>This step starts the Zowe instance</description>
+        <prereqStep name="define_variables"/>
+        <prereqStep name="create_instance"/>
+        <prereqStep name="change_instance_configuration"/>
+        <!--variableValues-->
+        <!-- condition -->
+        <instructions substitution="true">Run this step to start the Zowe instance.
+        
+If you want to start the instance manually (e.g. after IPL), you can do it with the following command:
+&lt;code&gt;sh "${instance-zowe_instance_path}/bin/zowe-start.sh"&lt;/code&gt;
+
+You can also start the instance using the following MVS command:
+
+&lt;code&gt;/S ZWESVSTC INSTANCE='${instance-zowe_instance_path}',JOBNAME=${instance-ZOWE_PREFIX}${instance-ZOWE_INSTANCE}SV&lt;/code&gt;
```

Comment:
the leading / in the command is specific for SDSF to route a command to the console. It should not be part of the sample, but we can add a comment saying 
Note: Operator commands can be issued from the SDSF command line if you are authorized to do so. In SDSF, operator commands are prefixed with the forward slash (/) character.

Reply:
I have added the SDSF related command and removed slash from the command example